src/cmd/manage_nsfs.js

```diff
@@ -476,6 +598,10 @@ async function validate_bucket_add_args(data) {
         console.error('Error: bucket path is mandatory, please use the --path flag');
         return false;
     }
+    if (!update && data.new_name) {
+        console.error('Error: Bucket new_name can not be used on add command, please remove the --new_name flag');
+        return false;
+    }
```

Comment:
We can maybe print a warning that we will ignor this flag and keep with the add,
@romayalon WDYT?

Reply:
that's what I initially wanted to do, but this may cause a confusion to the user, he won't understand why new_name is not there